app/src/main/java/com/humberto/tasky/agenda/presentation/agenda_details/mapper/DetailsMapper.kt

```diff
@@ -85,6 +94,27 @@ fun AgendaDetailsState.toAgendaItem(): AgendaItem {
     }
 }
 
+fun AttendeeUi.toAttendee(eventId: String, remindAt: ZonedDateTime): Attendee {
+    return Attendee(
+        userId = userId,
+        email = email,
+        fullName = fullName,
+        eventId = eventId,
+        isGoing = isGoing,
+        remindAt = remindAt
+    )
+}
+
+fun Attendee.toAttendeeUi(): AttendeeUi {
+    return AttendeeUi(
+        userId = userId,
+        email = email,
+        fullName = fullName,
+        isGoing = isGoing,
+        isAttendeeEventCreator = false
+    )
+}
```

Comment:
A static false? One of the attendees have to be the event creator, no? Could passing this information as a function parameter make more sense?